service/src/main/kotlin/fi/espoo/evaka/invoicing/data/FeeDecisionQueries.kt

```diff
@@ -156,7 +158,8 @@ private fun Database.Transaction.upsertDecisions(decisions: List<FeeDecision>) {
             head_of_family_income = :headOfFamilyIncome,
             partner_income = :partnerIncome,
             family_size = :familySize,
-            pricing = :pricing
+            pricing = :pricing,
+            sent_at = :sentAt
```

Comment:
Hmm why is this? Earlier this timestamping was done on a separate step / function call.